openslides_backend/http/views.py

```diff
@@ -89,11 +89,12 @@ def dispatch(self, body: RequestBody, headers: Headers) -> ResponseBody:
         payload: PresenterPayload = body
 
         # Handle request.
-        handler: Presenter = PresenterHandler()
+        handler: Presenter = PresenterHandler(
+            logging=self.logging, services=self.services,
+        )
+        user_id = "Test"
```

Comment:
See line 51 how to get the user id. this should already work and give an integer.

Reply:
Thanks. I moved the extraction part to the base class. See https://github.com/OpenSlides/openslides-backend/pull/40/commits/9e5bf7ff16b0b7375eaee4d92c6c4fcd6d21c35f